tools/tsp-client/src/commands.ts

```diff
@@ -284,6 +284,18 @@ export async function generateCommand(argv: any) {
   }
 }
 
+export async function compareCommand(argv: any, args: string[]) {
```

Comment:
also, could we add a test just to have a starting point to check that this command works as expected? Feel free to check in example files as needed